### DISCUSSION TITLE
I'm not able to link to headers

### DISCUSSION THREAD

**Discussion by frg100** [markdoc/markdoc#406] [Q&A] [answered]
### What happened?

I'm migrating our documentation to markdoc but I'm not able to link to sections. Is there anything special I have to do here?

eg header:
# Fact Table

eg link
when using a [Fact Table](#fact-table)

### To reproduce

I can't upload a snippet of my company's page, but I'm using the react renderer if that helps

### Version

0.3.0

**Comment by mfix-stripe** [comment, marked as answer]
Hey @frg100, you can see how we made our own docs (https://markdoc.dev) have linkable headers by looking at:

* https://github.com/markdoc/docs/blob/main/markdoc/nodes/heading.markdoc.js
and
* https://github.com/markdoc/docs/blob/main/components/Heading.js#L32
